multiversx_sdk_cli/dependencies/modules.py

```diff
@@ -347,14 +330,7 @@ def _get_env_for_is_installed_in_sdk(self) -> Dict[str, str]:
         }
 
     def get_env_for_install(self):
-        directory = self.get_directory("")
-
-        env = {
-            # For installation, wget (or curl) and cc (build-essential) are also required.
-            "PATH": f"{path.join(directory, 'bin')}:{os.environ['PATH']}",
-            "RUSTUP_HOME": str(directory),
-            "CARGO_HOME": str(directory)
-        }
+        env: Dict[str, str] = {}
```

Comment:
I think `get_env_for_install()` isn't used anymore? Also, `_get_env_for_is_installed_in_sdk()` - can both be removed?

Reply:
Alternatively, we can do the same as for `get_env()`, and keep the old calls to `get_env_for_install()`, but simply replace the implementation with `return os.environ`?